src/current/advisories/a104309.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Technical Advisory a104309
+advisory: A-a104309
+summary: In rare cases, a rangefeed bug may cause a checkpoint to be emitted prematurely in rare cases, before all writes below the checkpoint timestamp have been emitted.
+toc: true
+affected_versions: v1.0 to v22.2.17, v23.1.0 to v23.1.14, v23.2.0
```

Comment:
nit: did we have changefeeds in 1.0? Doesn't matter for practical purposes.

Reply:
Good call, they seem to have appeared in 2.1.11.